### PR TITLE
Fixes #23739: Add dependency support to rudder-package

### PR DESCRIPTION
**Pull request by Fdall**
https://issues.rudder.io/issues/23739

### CODE REVIEW

**Comment by Fdall** [comment]
PR updated with a new commit

**Comment by Fdall** [comment]
PR updated with a new commit

**Comment by Fdall** [comment]
PR updated with a new commit

File: relay/sources/rudder-package/src/cmd.rs [hunk 1]
```diff
@@ -0,0 +1,24 @@
+use std::str::FromStr;
```
Comment (amousset) [inline relay/sources/rudder-package/src/cmd.rs:1]: missing license header

**Comment by Fdall** [comment]
PR updated with a new commit

**Comment by Fdall** [comment]
PR updated with a new commit

**Pull request merged**